lib/modules/platform/types.ts

```diff
@@ -270,7 +270,10 @@ export interface Platform {
 
 export interface PlatformScm {
   isBranchBehindBase(branchName: string, baseBranch: string): Promise<boolean>;
-  isBranchModified(branchName: string): Promise<boolean>;
+  isBranchModified(
+    branchName: string,
+    baseBranch: string | undefined,
```

Comment:
```suggestion
    baseBranch?: string | undefined,
```

it should probably stay optional to have less code changes. so we don't need to explicit pass undefined

Reply:
In which case have we no baseBranch though? I would lean towards mandatory if there's no use case where we need told behavior